zkevm-circuits/src/state_circuit/constraint_builder.rs

```diff
@@ -320,19 +318,26 @@ impl<F: Field> ConstraintBuilder<F> {
     }
 
     fn build_tx_refund_constraints(&mut self, q: &Queries<F>) {
+        // 7.0. `address`, `field_tag` and `storage_key` are 0
         self.require_zero("address is 0 for TxRefund", q.rw_table.address.clone());
         self.require_zero("field_tag is 0 for TxRefund", q.field_tag());
         self.require_zero(
             "storage_key is 0 for TxRefund",
             q.rw_table.storage_key.clone(),
         );
-        self.require_zero("initial TxRefund value is 0", q.initial_value());
-
+        // 7.1. `state root` is not changed
         self.require_equal(
             "state_root is unchanged for TxRefund",
             q.state_root(),
             q.state_root_prev(),
         );
+        // 7.2. `initial value` is 0
+        self.require_zero("initial TxRefund value is 0", q.initial_value());
+        // 7.3. First access for a set of all keys are 0 if READ
```

Comment:
This constraint is redundant because on https://github.com/privacy-scaling-explorations/zkevm-circuits/blame/main/zkevm-circuits/src/state_circuit/constraint_builder.rs#L156, we already constrain `value` to be `initial_value`, which is 0 by 7.2 above.